src/custom-elements/src/components/popover/popover.tsx

```diff
@@ -55,6 +55,11 @@ export class Popover {
    */
   @Prop({ reflect: true }) reference: string;
 
+  /**
+   * To define portal style of the Popover
+   */
+   @Prop({ reflect: true }) portal: boolean;
```

Comment:
Please add a TODO comment to remove when all µApps update to Chi version higher than v5.0